site/en/docs/extensions/reference/alarms/index.md

```diff
@@ -2,39 +2,60 @@
 api: alarms
 ---
 
+## Manifest {: #manifest }
+
+To use `chrome.alarms` API, declare the `"alarms"` permission in the [manifest][doc-manifest]:
+
+```json
+{
+  "name": "My extension",
+  ...
+  "permissions": [
+    "alarms"
+  ],
+  ...
+}
+```
+
 ## Examples
 
 The following examples show how to use and respond to an alarm. To try this API,
 install the [Alarm API example][sample-example] from the [chrome-extension-samples][repo-samples]
 repository.
 
-### Set an alarm
+### Set an alarm {: #set-alarm }
 
-The following example sets an alarm in the service worker when the extension is installed, but not when it, Chrome or any shared module is updated.
+The following example sets an alarm in the service worker when the extension is installed, but not when it, Chrome, or any shared module is updated.
```

Comment:
Thoughts on this? I don't think the additional text is wrong but it seems unnecessary and confusing.

```suggestion
The following example sets an alarm in the service worker when the extension is installed:
```